Packaging/RedHat/nextspace_install.sh

```diff
@@ -45,13 +45,6 @@ if [ $EPEL_REPO != "" ]; then
     fi
 fi
 
-# More X drivers. Workaround until NextSpace RPMs include them as dependencies
-echo -n "Installing X11 drivers and utilities..."
-yum -y -q install xorg-x11-drivers xorg-x11-xinit xorg-x11-utils 2>&1 > /dev/null
-echo -e -n "\e[32m"
-echo "done"
-echo -e -n "\e[0m"
-
```

Comment:
Why have you removed this? Does it conflicts with SELinux in some way?

Reply:
Absolutely not - I removed it by mistake when changing the install script since I tested my snippet in isolation. It's completely independent of SELinux.

Reply:
Pushed